test_scripts/Polices/build_options/017_ATF_P_TC_PoliciesManager_Sets_Status_UPDATING_HTTP.lua

```diff
@@ -66,35 +66,37 @@ function Test:TestStep_PoliciesManager_changes_status_UPDATING()
   EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", {application = { appName = config.application1.registerAppInterfaceParams.appName } })
   :Do(function()
 
-      EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "HTTP"})
-      :Do(function()
-          if(message_order ~= 2) then
-            commonFunctions:printError("OnSystemRequest is not received as message 2 after OnAppRegistered. Received as message: "..message_order)
-            is_test_fail = true
-          else
-            print("OnSystemRequest received as message 2 after OnAppRegistered.")
+      EXPECT_NOTIFICATION("OnSystemRequest")--, {requestType = "LOCK_SCREEN_ICON_URL"}, {requestType = "HTTP"})
+      :Do(function(_,data)
+          print("SDL -> MOB: OnSystemRequest, requestType: " .. data.payload.requestType)
+          if(data.payload.requestType == "LOCK_SCREEN_ICON_URL") then
+            message_order = message_order + 1
           end
-          message_order = message_order + 1
-          local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest", { requestType = "HTTP", fileName = "PolicyTableUpdate", },"files/ptu.json")
-          EXPECT_RESPONSE(CorIdSystemRequest, { success = true, resultCode = "SUCCESS"})
-        end)
+          if(data.payload.requestType == "HTTP") then
+            if(message_order ~= 3) then
+              commonFunctions:printError("OnSystemRequest is not received as message 3 after OnAppRegistered. Received as message: "..message_order)
+              is_test_fail = true
+            end
+            message_order = message_order + 1
+            local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest", { requestType = "HTTP", fileName = "PolicyTableUpdate", },"files/ptu.json")
```

Comment:
@istoimenova, Please remove sending of SystemRequest since this action leads to completing of PTU sequence and getting of SDL.OnStatusUpdate(UP_TO_DATE) notification which is not expected in test step